api/admin/build.go

```diff
@@ -57,6 +59,54 @@ func AllBuilds(c *gin.Context) {
 	c.JSON(http.StatusOK, b)
 }
 
+// swagger:operation GET /api/v1/admin/builds/queue admin AllBuildsQueue
+//
+// Get all of the running and pending builds in the database
+//
+// ---
+// produces:
+// - application/json
+// parameters:
+// - in: query
+//   name: after
+//   description: Unix timestamp to limit builds returned
+//   required: false
+//   type: string
+// security:
+//   - ApiKeyAuth: []
+// responses:
+//   '200':
+//     description: Successfully retrieved all running and pending builds from the database
+//     schema:
+//       type: array
+//       items:
+//         "$ref": "#/definitions/BuildQueue"
+//   '500':
+//     description: Unable to retrieve all running and pending builds from the database
+//     schema:
+//       "$ref": "#/definitions/Error"
+
+// AllQueue represents the API handler to
```

Comment:
Could we update the comment here to reflect the function name?

```suggestion
// AllBuildsQueue represents the API handler to
```

Reply:
Fixed.